src/Card/Card.js

```diff
@@ -10,4 +11,12 @@ const Card = ( {name, artist, imageUrl, id, makeFavorite} ) => {
   )
 }
 
+Card.propTypes = {
+  id: PropTypes.string,
```

Comment:
[recommendation] an `isRequired` property or a `Card.defaultProps` object could give you even more error handling out of your prop types.